cmd/priv_val_server/main.go

```diff
@@ -30,7 +30,7 @@ func main() {
 		"privPath", *privValPath,
 	)
 
-	privVal := priv_val.LoadPrivValidatorJSON(*privValPath)
+	privVal := priv_val.LoadFilePV(*privValPath)
```

Comment:
mmmm priv_val.LoadFile,  or pvm.LoadFile I think is too generic.  I'd prefer to uniformly call it FilePV.

It could be called `pvm.LoadFromFile() types.PrivValidator`, but I think it's much better to be specific about what type of PV it is.